### PR TITLE
Adding plasticparcels

### PR DESCRIPTION
**Pull request by erikvansebille**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by michaeldenes** [comment]
I'd like to be listed here

**Comment by erikvansebille** [comment]
@conda-forge/help-python, ready for review!

File: recipes/plasticparcels/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{% set name = "plasticparcels" %}
+{% set version = "v0.1-alpha1" %}
```
[the remark this answers is not in the document]
Comment (ocefpaf) [reply recipes/plasticparcels/meta.yaml:2]: Also, that `v` is in the URL below and I'm guessing it is not part of the version number.

File: recipes/plasticparcels/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/oceanparcels/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 12ba5a969c8ec0a99fcd87d7db34698dd22394d52dc64a125212bfb3bb778a9c
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (ocefpaf) [inline recipes/plasticparcels/meta.yaml:13]:
```suggestion
  noarch: python
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

File: recipes/plasticparcels/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  number: 0
+
+requirements:
+  host:
+    - python
```
Comment (ocefpaf) [inline recipes/plasticparcels/meta.yaml:18]:
```suggestion
    - python >=3.8
```

File: recipes/plasticparcels/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    - pip
+  run:
+    - python
```
Comment (ocefpaf) [inline recipes/plasticparcels/meta.yaml:21]:
```suggestion
    - python >=3.8
```

File: recipes/plasticparcels/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,52 @@
+{% set name = "plasticparcels" %}
+{% set version = "0.1-alpha1" %}
```
Comment (ocefpaf) [inline recipes/plasticparcels/meta.yaml:2]: This is not pep440 and we cannot publish on conda-forge main unless it complies with CFEP-05.
PS: It is easier, IMO, to tag a stable version.

Comment (erikvansebille) [reply recipes/plasticparcels/meta.yaml]: Thanks @ocefpaf. We worked hard today and were able to release a new version, v0.1.0. See updated meta.yaml. Better like this?

**Pull request merged**